src/test/Components/Blueprints/Blueprints.test.tsx

```diff
@@ -167,22 +168,6 @@ describe('Blueprints', () => {
     ).not.toBeInTheDocument();
   });
 
-  test('CentOS 8 Stream renders', async () => {
```

Comment:
Shouldn't we keep this test? That should still work, right?
If there is a blueprint saved with centos-8, we should make sure we render some warning.